packages/e2e/package.json

```diff
@@ -0,0 +1,100 @@
+{
+    "name": "@cardano-sdk/e2e",
+    "version": "0.3.0",
+    "description": "End to end tests for the cardano-js-sdk packages.",
+    "main": "index.js",
+    "scripts": {
+        "test": "shx echo 'test' command not implemented yet",
+        "test:blockfrost": "jest -c jest.config.js --selectProjects blockfrost --runInBand --verbose",
+        "test:wallet": "jest -c jest.config.js --selectProjects wallet --runInBand --verbose",
+        "test:faucet": "jest -c jest.config.js --selectProjects faucet --runInBand --verbose",
+        "test:cardano-services": "jest -c jest.config.js --selectProjects cardano-services --runInBand --verbose",
+        "test:web-extension:build": "webpack -c test/web-extension/webpack.config.js",
+        "test:web-extension:run": "wdio run test/web-extension/wdio.conf.js",
+        "test:web-extension": "run-s test:web-extension:build test:web-extension:run",
+        "test:web-extension:watch:build": "yarn test:web-extension:build --watch",
+        "test:web-extension:watch:run": "yarn test:web-extension:run --watch",
+        "test:web-extension:watch": "run-s test:web-extension:build test:web-extension:watch:bg",
+        "test:web-extension:watch:bg": "run-p test:web-extension:watch:build test:web-extension:watch:run",
+        "private-network:up": "CARDANO_DB_SYNC_VERSION=13.0.0-rc3 DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1 OGMIOS_PORT=1338 NETWORK=testnet docker-compose -p private-testnet-e2e up",
```

Comment:
```suggestion
        "local-network:up": "CARDANO_DB_SYNC_VERSION=13.0.0-rc3 DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1 OGMIOS_PORT=1338 NETWORK=testnet docker-compose -p private-testnet-e2e up",
```
+ the other scripts

Reply:
fixed in https://github.com/input-output-hk/cardano-js-sdk/pull/305/commits/d963d0afe05db27b88d1451925c08e851f749877